src/microbe_stage/systems/EngulfingSystem.cs

```diff
@@ -186,6 +187,17 @@ protected override void Update(float delta, in Entity entity)
             return;
         }
 
+        // Not full anymore tutorial (end trigger for engulfment full tutorial)
+        if (wasFullAlready && engulfer.UsedEngulfingCapacity <= engulfer.EngulfingSize * 0.5f)
```

Comment:
Yet to be confirmed that 50% is a good threshold.

Reply:
Is there also a time requirement before closing? I'd make sure the tutorial is visible for at least 15 seconds to ensure the player has time to read it.

Also there is a real github issue about this, not just a roadmap item: https://github.com/Revolutionary-Games/Thrive/issues/3565